packages/cli/src/commands/tile/create.ts

```diff
@@ -0,0 +1,61 @@
+import { flags } from '@oclif/command'
+import { TileDocument } from '@ceramicnetwork/stream-tile'
+import type { TileMetadataArgs } from '@ceramicnetwork/stream-tile'
+
+import { Command } from '../../command'
+import type { CommandFlags } from '../../command'
+
+type Flags = CommandFlags & {
+  'only-genesis'?: boolean
+  metadata?: TileMetadataArgs
+}
+
+export default class CreateTile extends Command<Flags, { content: string }> {
+  static description = 'create a new Tile stream'
+
+  static args = [
+    {
+      name: 'content',
+      description: 'stream contents (JSON encoded as string)',
+      required: true,
+      parse: JSON.parse,
+    },
+  ]
+
+  static flags = {
+    ...Command.flags,
+    'only-genesis': flags.boolean({
+      char: 'g',
+      description: 'only generate genesis commit',
+      default: false,
+    }),
+    metadata: flags.string({
+      char: 'm',
+      description: 'stream metadata',
+      parse: JSON.parse,
+    }),
+  }
+
+  async run(): Promise<void> {
+    this.spinner.start('Creating stream...')
+    try {
+      const metadata = this.flags.metadata ?? {}
+      metadata.controllers = [this.authenticatedDID.id]
+
+      const tile = await TileDocument.create(this.ceramic, this.args.content, metadata, {
+        anchor: !this.flags['only-genesis'],
```

Comment:
ah I see now what `only-genesis` does.  I feel like this isn't very intuitive.  Why would you ever want to set this flag?  Should we instead have separate `--no-anchor` and `--no-publish` flags?  Or just remove this option entirely?

Reply:
Yeah let's remove these if you don't think it's useful.

Reply:
Removing now.